modules/token-holders/pages/custody-options-intro.adoc

```diff
@@ -0,0 +1,84 @@
+:sdk-long-name: DFINITY Canister Software Development Kit (SDK)
+ifdef::env-github,env-browser[:outfilesuffix:.adoc]
+
+Self-custody is a token distribution option that enables you to have sole responsibility over your digital assets and over the management of your private keys. 
+With self-custody, you have complete and direct control over access to your digital assets, rather than using a third-party to act on your behalf.
+
+If you have made the choice to control your digital assets yourself, you should be aware that there are different kinds of *self-custody solutions* and that those solutions vary in their security and ease of use. 
+Before you decide on a specific self-custody solution, you should spend some time learning about the various options available and how they differ. 
+
+This guide provides a brief introduction to the concept of *cold storage* as part of your custody strategy and the most common types of cold storage currently available. 
+
+For additional information about cold storage and other custody options, you should do your own research and be sure that you understand the ramifications of your choices thoroughly before proceeding.
+
+== Cold storage
+
+Cold storage means that you are keeping private keys stored offline in a device that is not connected to the internet. 
+Cold storage is a popular solution for self-custody of digital assets because it makes it more difficult for your keys to be compromised if there is a software security breach or if your cold storage solution is lost or stolen.
+
+For more information about approaches to cold storage and best practices for securing your private keys, see the following additional resources:
+
+* link:https://www.investopedia.com/terms/c/cold-storage.asp[Cold storage]
+
+* link:https://www.ledger.com/academy/hardwarewallet/best-practices-when-using-a-hardware-wallet#:~:text=%E2%80%93%20Keep%20your%20Recovery%20sheet%20physically,or%20smartphone%20screen%20with%20caution[Best Practices when using a Hardware Wallet]
+
+* link:https://blog.trailofbits.com/2018/11/27/10-rules-for-the-secure-use-of-cryptocurrency-hardware-wallets/[10 Rules for the Secure Use of Cryptocurrency Hardware Wallets]
+
+== Common cold storage options
+
+The most common types of cold storage include the following:
+
+* <<Hardware wallets>>
+* <<Air-gapped computers>>
+* <<Paper wallets>>
+
+=== Hardware wallets
+
+A *hardware wallet* is a physical device that holds your digital assets. 
+Private keys are stored in a secure part of the device that does not come into contact with the internet. 
+The partitioned storage on the physical device reduces the risks that are associated with devices connected to the internet. 
+Because you are relying on a physical device, however, losing the device also means losing access to your digital assets.
+
+In most cases, you can recover from the loss of a hardware wallet because the assets stored on a hardware wallets are secured using a secret seed phase
```

Comment:
```suggestion
In most cases, you can recover from the loss of a hardware wallet because the assets stored on hardware wallets are secured using a secret seed phrase.
```